tests/functional/storage/test_s3_functional.py

```diff
@@ -42,6 +42,24 @@ def test_s3_storage(funcx_s3_bucket):
     assert task.result_reference["storage_id"] == "s3"
 
 
+@pytest.mark.skipif(not has_boto, reason="Test requires boto3 lib")
```

Comment:
The `funcx_s3_bucket` fixture returns `None` if the env var isn't set. These tests should be skipped in that case.

Reply:
Yes. I also think the skip should be done in an autouse fixture, so that we don't have to mark every test in the module in the same way. I'll make the change.

Reply:
I just went to do this and the autouse fixture is already in place. These tests will be skipped if that fixture isn't in place.

I'm still going to make a change to put the `not has_boto` skip into module scope, but there's no functional change to make here.